src/commands/playercount.ts

```diff
@@ -15,10 +15,41 @@ const PlayerCountCommand: Command = {
         content: "Failed to get server data",
         ephemeral: true,
       });
+      return;
+    }
+
+    let uptimeStr = "";
+    if (data.uptime.serverStartUnix) {
+      const diff = Math.floor(Date.now() / 1000 - data.uptime.serverStartUnix);
+      const days = Math.floor(diff / 86400);
+      const hours = Math.floor((diff % 86400) / 3600);
+      const minutes = Math.floor((diff % 3600) / 60);
+      const seconds = diff % 60;
+      const parts = [];
+      if (days > 0) parts.push(`${days}d`);
+      if (hours > 0) parts.push(`${hours}h`);
+      if (minutes > 0) parts.push(`${minutes}m`);
+      if (seconds > 0) parts.push(`${seconds}s`);
+      uptimeStr = parts.join(" ");
+    } else if (data.uptime.currentUptimeHours) {
+      const totalSeconds = Math.floor(data.uptime.currentUptimeHours * 3600);
+      const days = Math.floor(totalSeconds / 86400);
+      const hours = Math.floor((totalSeconds % 86400) / 3600);
+      const minutes = Math.floor((totalSeconds % 3600) / 60);
+      const seconds = totalSeconds % 60;
+      const parts = [];
+      if (days > 0) parts.push(`${days}d`);
+      if (hours > 0) parts.push(`${hours}h`);
+      if (minutes > 0) parts.push(`${minutes}m`);
+      if (seconds > 0) parts.push(`${seconds}s`);
+      uptimeStr = parts.join(" ");
+    } else {
+      await interaction.reply(`The server is currently down.`);
+      return;
     }
```

Comment:
there has got to be a better way to do this rather than to duplicate it twice.

maybe use some library
maybe use a shared function
preferably a library